src/lib/mongoBackend/mongoSubCache.cpp

```diff
@@ -1056,9 +1069,17 @@ void mongoSubCacheStatisticsGet(int* refreshes, int* inserts, int* removes, int*
   {
     while (cSubP != NULL)
     {
+      char msg[256];
+
       unsigned int bytesLeft = listSize - strlen(list);
 
-      if (strlen(cSubP->subscriptionId) + 2 > bytesLeft)
+#if UpdateBugTests
```

Comment:
Yeah, not really a leftover, see I need this trace (even if it is temporal) to catch the "possible" bug about updating subcsriptions in the cache